coordinator/src/main/scala/filodb.coordinator/NodeCoordinatorActor.scala

```diff
@@ -182,7 +182,7 @@ class NodeCoordinatorActor(metaStore: MetaStore,
   }
 
   private def reloadDatasetCoordActors(originator: ActorRef) : Unit = {
-    logger.debug(s"Reload of dataset coordinator actors has started for path: $actorPath")
+    logger.info(s"Reload of dataset coordinator actors has started for path: $actorPath")
```

Comment:
I think this is good to move to info, but for all others can be kept at debug, ok?